tests/unit/type/list.tcl

```diff
@@ -2005,4 +2029,7 @@ foreach {pop} {BLPOP BLMPOP_RIGHT} {
         assert_equal [lpop k] [string repeat x 31]
         set _ $k
     } {12 0 9223372036854775808 2147483647 32767 127}
+    config_set list-max-listpack-size $origin_config
+}
+
 }
```

Comment:
If indentation isn't really correct, it can be good to add a comment about what each brace is matching.

    } ; # foreach type

Reply:
Done